lacework/resource_lacework_integration_ecr.go

```diff
@@ -33,14 +71,32 @@ func resourceLaceworkIntegrationEcr() *schema.Resource {
 				Type:     schema.TypeString,
 				Required: true,
 			},
-			"access_key_id": {
-				Type:     schema.TypeString,
+			"credentials": {
+				Type:     schema.TypeList,
+				MaxItems: 1,
 				Required: true,
-			},
-			"secret_access_key": {
```

Comment:
We will stop supporting these arguments at the root of the resource and therefore we will bump the version to `0.2.0`. Thought should we be bumping to `1.0.0`since this change is not backward compatible?